stock_scanner_inventory/__init__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Thanks, I emptied the __init__.py file, which can be empty for this type of module (no licence needed for an empty file, I think).